src/App.js

```diff
@@ -8,10 +8,10 @@ import './App.css';
 class App extends Component {
   render() {
     return (
-      <div class="presentation animated fadeIn slow">
+      <div className="presentation animated fadeIn slow">
```

Comment:
I think this is not correctly indented, use some tool to make it automatic. Also, you can add this code format tasks before your commit or your push process (pre-commit or pre-push hooks)